src/main/java/io/gravitee/connector/http/AbstractHttpConnector.java

```diff
@@ -120,6 +122,8 @@ public void request(ExecutionContext context, ProxyRequest request, Handler<Conn
             if (endpoint.getHeaders() != null && !endpoint.getHeaders().isEmpty()) {
                 endpoint
                     .getHeaders()
+                    .stream()
+                    .filter(httpHeader -> !httpHeader.getName().equalsIgnoreCase(HttpHeaders.ACCEPT_ENCODING))
```

Comment:
don't remove this header from endpoint header list, but from the incoming request